packages/eslint-plugin/src/rules/no-empty-interface.ts

```diff
@@ -19,6 +19,7 @@ export default createRule<Options, MessageIds>({
     deprecated: true,
     docs: {
       description: 'Disallow the declaration of empty interfaces',
+      recommended: 'recommended',
```

Comment:
Heh, I'd wanted to show the failure in CI, but https://github.com/typescript-eslint/typescript-eslint/actions/runs/12564281988/job/35027242811?pr=10583 passed even with the failure. 🤷 

```plaintext
Error: no-empty-interface is both deprecated and recommended.
    at reducer (/home/runner/work/typescript-eslint/typescript-eslint/tools/scripts/generate-configs.mts:124:15)
    at <anonymous> (/home/runner/work/typescript-eslint/typescript-eslint/tools/scripts/generate-configs.mts:303:30)
    at Array.reduce (<anonymous>)
    at getConfig (/home/runner/work/typescript-eslint/typescript-eslint/tools/scripts/generate-configs.mts:302:28)
    at writeConfig (/home/runner/work/typescript-eslint/typescript-eslint/tools/scripts/generate-configs.mts:197:20)
    at writeExtendedConfig (/home/runner/work/typescript-eslint/typescript-eslint/tools/scripts/generate-configs.mts:298:11)
    at main (/home/runner/work/typescript-eslint/typescript-eslint/tools/scripts/generate-configs.mts:332:9)
```

```suggestion
```